### PR TITLE
fix : Remove menu icons from map fragment

### LINKED ISSUE #1752: Menu icons are visible on showing map fragment
**Actual Behaviour**

1. In session detail activity and location activity on clicking map icon map fragment is shown but menu icons(session search, share, map) still appears. Because of it user can again click on map fragment and so on. IMO if map fragment is shown then menu items(session search, share, map) should not appear. Also tooldar should display location name instead of session name.

2. In location activity after clicking on location and map icon, on pressing back button sessions for location is disappears,

**Expected Behaviour**

- If user clicks on map icon then menu icons should be disappear.
- Sessions should be visible in location activity after pressing back button in map fragment.

**Steps to reproduce issue**
1. First issue
- Go to session detail activity / location activity
- click on map icon
- Menu icons still appears

2. Second issue
- Go to location fragment
- click on any location
- Click on map icon 
- press back button
- session disappears

**Would you like to work on the issue?**

Yes

**Target Milestone to solve the issue**

@iamareebjamal  Please add to [Milestone 8](https://github.com/fossasia/open-event-android/milestone/31)

### PR DESCRIPTION
**Pull request by Shailesh351**
Fixes #1752 

Changes: 
- Removed menu icons from session details activity and locat
tion activity
- Changed toolbar title from session name to location name when showing map fragment

Screenshots for the change: 

videotogif_2017 06 30_19 51 16

### CODE REVIEW

**Comment by Shailesh351** [comment]
@iamareebjamal @arp95 @rohanagarwal94 @shreyasnbhat Please review

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java [hunk 1]
```diff
@@ -153,7 +155,10 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public void onBackPressed(){
         if((mSessions.isEmpty())){
             noSessionsView.setVisibility(View.VISIBLE);
+        } else {
+            sessionRecyclerView.setVisibility(View.VISIBLE);
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java:159]: Both conditions are `VISIBLE`

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -85,9 +85,8 @@
         <item name="android:orientation">vertical</item>
     </style>
 
-    <style name="ExpandedAppBar" parent="@android:style/TextAppearance.Medium">
-        <item name="android:textSize">20sp</item>
-        <item name="android:textColor">@android:color/holo_green_dark</item>
+    <style name="ExpandedAppBar">
+        <item name="android:textSize">22sp</item>
```
Comment (arp95) [inline android/app/src/main/res/values/styles.xml]: please use the value from dimens.xml

**Review by arp95 — APPROVED** [review]
LGTM.

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java (hunk 1 above)
Comment (Shailesh351) [reply android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java:159]: Yes, but one is for noSessionsView and second is for sessionRecyclerView. 
On clicking **map icon** we are setting visibility of both to GONE. So on pressing back button one of them should be visible

Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java:159]: OK

**Review by iamareebjamal — DISMISSED** [review]

**Review by shreyasnbhat — APPROVED** [review]

**Comment by rohanagarwal94** [comment]
@shailesh351 please squash commits.

**Pull request merged**